docs/index.rst

```diff
@@ -3,6 +3,370 @@
+  {"f1": "value3"}
+
+The three records entered will be published to the Kafka topic ``test_hdfs`` in Avro format.
+
+Next, in ``./config/connect-standalone.properties``, we need to use
+``io.confluent.connect.avro.AvroConverter`` for ``key.converter`` and ``value.converter`` and
+provide the schema registry urls for them::
+
+  key.converter=io.confluent.connect.avro.AvroConverter
+  key.converter.schema.registry.url=http://localhost:8081
+  value.converter=io.confluent.connect.avro.AvroConverter
+  value.converter.schema.registry.url=http://localhost:8081
+
+Now create a configuration file ``quickstart-hdfs.properties`` for the HDFS connector::
+
+  name=hdfs-sink
+  connector.class=io.confluent.connect.hdfs.HdfsSinkConnector
+  tasks.max=1
+  topics=test_hdfs
+  hdfs.url=hdfs://localhost:9000
+  flush.size=3
+
+The first few settings are common settings you'll specify for all connectors. The ``topics``
+specifies the topics we want to export data from, in this case ``test_hdfs``. The ``hdfs.url``
+specifies the HDFS we are writing to and you should set this according to your configuration. The
+``flush.size`` specifies the number of records we see before we commit the file to HDFS.
+
+Then run the following command to start Kafka connect with the HDFS connector::
+
+  $ ./bin/connect-standalone ./config/connect-standalone.properties quickstart-hdfs.properties
+
+You should see the process start up and log some messages, and then it will export data from
+Kafka to HDFS. In order to check that the data have been copied to HDFS, you can use Hive to query
+data or Avro tools to extract the file in HDFS::
+
+  $ hadoop fs -ls /topics/test_hdfs/partitions=0
+
+You should see a file with name ``/topics/t1/partition=0/test_hdfs+0+0000000000+0000000002.avro``
+The file name is encoded as ``topic+partition+start_offset+end_offset.format``.
+
+You can use ``avro-tools-1.7.7.jar`` to extract the content of the file::
+
+  $ hadoop jar avro-tools-1.7.7.jar tojson /topics/test_hdfs/partition=0/test_hdfs+0+0000000000+0000000002.avro
+
+Features
+--------
+The HDFS connector offers a bunch of features as follows:
+
+* **Exactly Once Delivery**: The connector uses a write ahead log to make sure each record exports
+  to HDFS exactly once. Also, the connector manages offset commit by encoding the Kafka offset
+  information into the file so that the we can start from the last committed offset in case of
+  failures and task restart.
+
+* **Extensible Data Format**: Out of the box, the connector supports writing data to HDFS in Avro
+  and Parquet format. Also, you can write other formats to HDFS by extending the ``Format`` class.
+
+* **Hive Integration**: The connector supports Hive integration out of the box, and when it is
+  enabled, the connector automatically creates a Hive external partitioned table for each topic
+  exported to HDFS.
+
+* **Schema Evolution**: The connector supports schema evolution and different schema compatibility.
```

Comment:
Can be a follow up, but we need a lot more explanation on what the different settings mean (with and without Hive).
I remember you wrote documentation about it, so perhaps you have something you can just copy-paste here?